framework/decode/openxr_replay_consumer_base.cpp

```diff
@@ -801,7 +823,7 @@ static EventStrings events_to_string[] = {
 const char* GetEventTypeString(XrStructureType type)
 {
     size_t len = sizeof(events_to_string) / sizeof(EventStrings);
-    for (size_t ii = 0; ii < len; ++ii)
+    for (size_t ii = 1; ii < len; ++ii)
```

Comment:
Sorry, I think I fouled this up.  For "Get.*String" we do need entry 0.  I thought this was for the `IsEventHandled` which seems to have disappeared since the last time I looked at this more carefully.